src/api/advanced.js

```diff
@@ -54,16 +58,17 @@ export function getDelayedList(params) {
   return http.get('/mqtt/delayed/messages', { params: params })
 }
 
-export function getDelayedInfo(id) {
-  if (id == null) return
-  return http.get('/mqtt/delayed/messages/' + encodeURIComponent(id))
+export function getDelayedInfo(node, id) {
+  if (!node || id == null) return
```

Comment:
Although `==` does not affect the correctness here, but I still recommend it more `===`